docs/application-connector/docs/08-02-get-client-certificate.md

```diff
@@ -100,15 +102,49 @@ The response contains a valid client certificate signed by the Kyma Certificate
 
 After you receive the certificate, decode it and use it in your application. Register the services of your external solution through the Application Registry.
 
-## Call the Metadata and Event services on local deployment
+## Call metadata endpoint
+
+You must call the metadata endpoint with the generated certificate to get URLs to the following:
+
+- Application Registry API
+- Events Service API
+- Certificate renewal endpoint
+- Certificate revocation endpoint
+
+Use the link you got in the second step to fetch the metadata information. Run:
```

Comment:
Maybe providing the exact link would be a better idea?

Reply:
Done